slides/final_slides.md

```diff
@@ -1,11 +1,27 @@
 % Project Gamma Progress Report
```

Comment:
The conflicts in this file need to be resolved. Whenever you see <<<<< HEAD, there is a conflict between your local version and the repo test branch version. You can call me for questions when you are about to resolve them.